ta4j-core/src/test/java/org/ta4j/core/indicators/ChopIndicatorTest.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * 
+ */
+package org.ta4j.core.indicators;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.time.ZonedDateTime;
+import java.util.Random;
+import java.util.function.Function;
+import org.junit.jupiter.api.Test;
+import org.ta4j.core.BaseTimeSeries;
+import org.ta4j.core.Indicator;
+import org.ta4j.core.TimeSeries;
+import org.ta4j.core.mocks.MockTimeSeries;
+import org.ta4j.core.num.BigDecimalNum;
+import org.ta4j.core.num.DoubleNum;
+import org.ta4j.core.num.Num;
+
+/**
+ * @author jtomkinson
+ *
+ */
+class ChopIndicatorTest  {
+	/**
+	 * this will assert that choppiness is high if market price is not moving
+	 */
+	@Test
+	void testChoppy() {
+        BaseTimeSeries.SeriesBuilder timeSeriesBuilder = new BaseTimeSeries.SeriesBuilder();
+        TimeSeries series = timeSeriesBuilder.withName("low volatility series").withNumTypeOf(DoubleNum::valueOf).build();

```

Comment:
After extending the `AbstractIndicatorTest ` you can generate `TimeSeries ` using **numFunction**:
```java
timeSeriesBuilder.withName("low volatility series").withNumTypeOf(numFunction).build(); //numFunction field of AbstractIndicatorTest
```